e2e/evm-evm/docker-compose.e2e.yml

```diff
@@ -0,0 +1,126 @@
+# Copyright 2020 ChainSafe Systems
+# SPDX-License-Identifier: LGPL-3.0-only
+version: '3'
+
+services:
+  evm1-1:
```

Comment:
Why do we need 6 evm instances if we are only testing on 2?

Reply:
yeap, definitely no need to have 3 nodes per network, fixed